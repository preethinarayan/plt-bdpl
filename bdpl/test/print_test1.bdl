print("This is a test print::Done printing");
print("This should not be on a new line::");
print("This should be on a new line");