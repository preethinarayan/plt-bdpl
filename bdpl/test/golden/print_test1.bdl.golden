This is a test print
Done printingThis should not be on a new line
This should be on a new line